data/parquet/README.md

```diff
@@ -21,4 +21,5 @@
 
 | File | Description |
 | --- | --- |
-| ARROW-17100.parquet | Parquet file written by PyArrow 2.0 with DataPageV2 and compressed columns. Prior to PyArrow 3.0, pages were compressed even if the is_compressed flag was 0. This was fixed in ARROW-10353, but for backwards compatibility readers may wish to support such a file. |
\ No newline at end of file
+| ARROW-17100.parquet | Parquet file written by PyArrow 2.0 with DataPageV2 and compressed columns. Prior to PyArrow 3.0, pages were compressed even if the is_compressed flag was 0. This was fixed in ARROW-10353, but for backwards compatibility readers may wish to support such a file. |
+| alltypes-java.parquet | Parquet file written by using the Java DatasetWriter class in Arrow 14.0. Types supported do not include Map, Sparse and DenseUnion, Interval Day, Year, or MonthDayNano, and Float16 |
```

Comment:
Can we link to PR https://github.com/apache/arrow/pull/38249 and/or the class that generated it?

Reply:
Done.